controller/index.js

```diff
@@ -9,16 +9,40 @@ The information taken from the example postman would be the email,
 name of person, and phone number right?
 */
 
-const getAllContacts = (request, response) => {
-  return response.status(200).send('This was successful')
+const getAllContacts = async (request, response) => {
```

Comment:
I will go over this today. But these contact functions should go on their own controller file. The name of the file could be something like `controller/contactsController.js`.
You have to see contact as its own entity. This will help you group the functions together that belong in this case to contacts. If we added all the functions in this file(for example the contact functions, the text message functions, the order functions, ...), this file would become very large and hard to maintain.